### PR TITLE
Setup translation infrastructure

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: locale/bn/LC_MESSAGES/hammer-cli-katello.mo
Comment (chris1984) [inline locale/bn/LC_MESSAGES/hammer-cli-katello.mo:1]: Are we wanting to keep the .mo files in the repo? I have heard mixed things about either keeping it out of the repo and just including it when you make the gem, or some repos have the mo files in them. I don't care either way I just want to know what the correct way is?

**Review by chris1984 — COMMENTED** [review]
2 small questions but non blocking.

File: .tx/config [hunk 1]
```diff
@@ -6,5 +6,4 @@ file_filter = locale/<lang>/hammer-cli-katello.edit.po
 source_file = locale/hammer-cli-katello.pot
 source_lang = en
 type = PO
-minimum_perc  = 0
```
[the remark this answers is not in the document]
Comment (ofedoren) [reply .tx/config:9]: Well, it doesn't make much sense to have empty files. Also, per @ekohl, it's better if we include >= 50% only.

**Comment by ofedoren** [comment]
As far as I know, there is no rush with this.

**Pull request merged**